turbograft.gemspec

```diff
@@ -6,7 +6,7 @@ require 'turbograft/version'
 Gem::Specification.new do |spec|
   spec.name          = "turbograft"
   spec.version       = TurboGraft::VERSION
-  spec.authors       = ["Kristian Plettenberg-Dussault", "Justin Li", "Nicholas Simmons", "Tyler Mercier", "Anthony Cameron", "Patrick Donovan"]
+  spec.authors       = ["Kristian Plettenberg-Dussault", "Justin Li", "Nicholas Simmons", "Tyler Mercier", "Anthony Cameron", "Patrick Donovan", "Mathew Allen"]
   spec.email         = ["tylermercier@gmail.com"]
   spec.summary       = "turbolinks with partial page replacement"
   spec.description   = "It's like turbolinks, but with partial page replacement and tests"
```

Comment:
OMG YOU FOUND IT

I noticed our overly smarmy description on rubygems and it made me sad but I couldn't figure out where it was coming from. Never occurred to me it would be in the gemspec. :{